docs/code-signing.rst

```diff
@@ -0,0 +1,10 @@
+Code signing policy
+===================
+
+The code used by the DebOps Project and commited in its repositories is
+expected to be signed by a valid GPG key of a DebOps Developer or DebOps
+Contributor. This should allow for secure code authentication and a valid audit
```

Comment:
> more about someone bringing in changes which will blow my setup. Therefore I need to do a manual code audit anyway before rolling out any changes to my environment.

@ganto So I guess you are reviewing parts of DebOps already? Maybe it makes sense to take a similar approach as me who also does this by additionally signing the latest version of DebOps reviewed components using git submodules and GPG. This way, when someone trusts you, they can infer some of that trust onto the reviewed components. Or when multiple people do this who can somehow be trusted or are expected to don’t cooperate or the like that can also build up some trust or at least make it more difficult to to introduce flaws and not get caught.

I recently documented this a bit more: https://github.com/ypid/ypid-ansible-common/